lsp/src/handle.rs

```diff
@@ -68,6 +68,10 @@ fn handle_completion(req: Request) -> Option<HtmxResult> {
         Some(CompletionContext {
             trigger_kind: CompletionTriggerKind::TRIGGER_CHARACTER,
             ..
+        })
+        | Some(CompletionContext {
+            trigger_kind: CompletionTriggerKind::INVOKED,
```

Comment:
Looks like VSC*de sends INVOKED instead of TRIGGER_CHARACTER for HTML -'s

Related: https://github.com/microsoft/vscode/issues/130953

Reply:
i now have - and " as my required characters! but lets start here